src/modules/extendedCallWindow/i18n/fi_FI.json

```diff
@@ -12,7 +12,7 @@
         }
     },
     "arrHighlight": {
-        "removeBeforeParsing": ["(PALOHÄLYTYSJÄRJESTELMÄ)", "[LIITTOUMA]"]
+        "removeBeforeParsing": [" (PALOHÄLYTYSJÄRJESTELMÄ)", " [LIITTOUMA]"]
```

Comment:
This doesn't seem correct. There is no reason why there should be space at the start.